app/src/release/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,20 @@
+<resources>
```

Comment:
Should this file be here?

Reply:
This one is in the release folder: i'll do the same thing as for the debug folder API key. You won't have to mess with it unless you're building in release.

Reply:
sounds good